source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/lib/index.ts

```diff
@@ -89,9 +105,32 @@ export class LambdaToElasticSearchAndKibana extends Construct {
     super(scope, id);
     defaults.CheckProps(props);
 
+    if ((props.deployVpc || props.vpcProps || props.existingVpc) &&
+        (props.lambdaFunctionProps && (props.lambdaFunctionProps.vpc || props.lambdaFunctionProps.vpcSubnets))) {
+      throw new Error("Error - More than 1 VPC specified for Lambda function");
+    }
+
+    if ((props.deployVpc || props.vpcProps || props.existingVpc) &&
+        props.esDomainProps && props.esDomainProps.vpcOptions) {
+      throw new Error("Error - More than 1 VPC specified in the properties");
+    }
+
+    if (props.deployVpc || props.existingVpc) {
+      this.vpc = defaults.buildVpc(scope, {
```

Comment:
Missing this check here:

```
  if (props.deployVpc && props.existingVpc) {
          throw new Error("More than 1 VPC specified in the properties");
        }
```

Reply:
This check is covered in `defaults.CheckProps(props)`, should add one more check here?